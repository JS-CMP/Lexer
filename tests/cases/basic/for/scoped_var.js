for (var i = 0; i < 3; i++) {
    var x = i * 2;
    console.log(x);
}
// console.log(x); Not supported by the lib and the lexer