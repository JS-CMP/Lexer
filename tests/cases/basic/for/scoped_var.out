#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {
    JS::Any i, x;
    for (i = JS::Any(0); i < JS::Any(3); i++) 
    {
        x = i * JS::Any(2);
        console[u"log"](x);
    }
    return 0;
}
